sugoi-api-core/src/main/java/fr/insee/sugoi/core/store/WriterStore.java

```diff
@@ -24,41 +24,181 @@
 /** Writer stores are responsible for all operations modifying the underlying store. */
 public interface WriterStore {
 
-  void deleteUser(String id);
-
+  /**
+   * Create the user in the store.
+   *
+   * @param user
+   * @return the user as it has been passed (address location migth have been added).
+   */
   User createUser(User user);
 
+  /**
+   * Replace the user with the same id by the updatedUser in the store.
+   *
+   * @param updatedUser
+   * @return the updatedUser as it has been passed (address location migth have been added).
+   */
   User updateUser(User updatedUser);
 
-  void deleteGroup(String appName, String groupName);
+  /**
+   * Delete the user id in the store.
+   *
+   * @param organizationId
+   */
+  void deleteUser(String id);
 
-  Group createGroup(String appName, Group group);
+  /**
+   * Create the organization in the store. Throw UnsupportedOperationException if the configuration
```

Comment:
There is a `@throws` annotation which could be useful here

Reply:
is it not only for checked exceptions ?

Reply:
but maybe we do want the same behavior than a checked exception but without having to try catch

Reply:
not sure : https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#throwstag

> The purpose of the @throws tag is to indicate which exceptions the programmer must catch (for checked exceptions) or might want to catch (for unchecked exceptions).

We don't want to mention every exception possible but here UnsupportedOperationException is a kind of functionality :)

Reply:
ok thanks :) I changed it